components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2ScopeService.java

```diff
@@ -258,16 +258,16 @@ public Scope updateScope(Scope updatedScope) throws IdentityOAuth2ScopeException
      * @return List of available scopes
      * @throws IdentityOAuth2ScopeServerException
      */
-    private Set<Scope> listScopesWithPagination(int startIndex, int count)
+    private Set<Scope> listScopesWithPagination(Integer startIndex, Integer count)
```

Comment:
Why was this changed from int to Integer?

Reply:
Because the method which is calling listScopeWithPagination is passing Integer object[1] so there is a possibility of NPE. 

[1]https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/30b637141b1f930f2aa89c9a2f98d08c8d711735/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2ScopeService.java#L105